console/src/components/Main/Main.js

```diff
@@ -306,6 +307,16 @@ class Main extends React.Component {
                         <span>Reach out ({'support@hasura.io'})</span>
                       </a>
                     </li>
+                    <li className={'dropdown-item'}>
+                      <a href="https://docs.hasura.io/1.0/graphql/manual/index.html">
```

Comment:
```suggestion
                      <a href="https://docs.hasura.io/">
```

Reply:
Will be better so that it's redirected to the newest version of the docs always.